.travis.yml

```diff
@@ -1,40 +1,52 @@
+---
 language: rust
 os:
-    - linux
-    - osx
+  - linux
+  - osx
 rust:
-    - stable
-    - beta
-    - nightly
+  - nightly
+  - beta
+  - stable
+  - 1.32.0  # pinned minimum supported toolchain
+
 matrix:
-    allow_failures:
-        - rust: nightly
+  allow_failures:
+    - rust: nightly
+  include:
+    - name: "Lints with stable toolchain"
+      os: linux
+      arch: amd64
+      rust: stable
+      before_script:
+        - rustup component add clippy
+        - rustup component add rustfmt
+      script:
+        # Ideally we would run CLippy on all targets, but benchmarks depend
+        # on an unstable feature and only work on nightly.
+        # - cargo clippy --all-targets -- -D warnings
+        - cargo fmt -- --check
```

Comment:
Nit: it'd be clearer if this line was before the comment (as before), given that the comment relates to Clippy.